src/Parser/Shaman/Elemental/Modules/Talents/PrimalStormElemental.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+
+const damagingCasts = [SPELLS.EYE_OF_THE_STORM.id, SPELLS.WIND_GUST.id, SPELLS.CALL_LIGHTNING.id];
+const CALL_LIGHTNING__BUFF_DURATION = 15000;
+
+class PrimalStormElemental extends Analyzer {
+  eotsCasts = 0;
+  pseCasts = 0;
+  lastCLCastTimestamp = 0;
+
+
+  damageGained = 0;
+  maelstromGained = 0;
+  badCasts=0;
```

Comment:
eotsCasts, pseCasts, maelstromGained and badCasts are never used. lastCLCastTimestamp is never set.

Reply:
Same thing as Primal Fire Elemental.
I can remove the variables for now and readd them in a later PR.